src/.eslintrc.json

```diff
@@ -11,7 +11,26 @@
     "no-unused-vars": "off",
     "func-names": ["warn", "never"],
     "no-eq-null": "off",
-    "eqeqeq": ["error", "smart"]
+    "eqeqeq": ["error", "smart"],
+    "no-underscore-dangle": "off",
+    "strict": "off",
```

Comment:
why not strict?

Reply:
With strict I have to write "use-strict" on the top of basically every file, I don't think it's that needed, when eslint and the tests do the same job. Too much spam IMO.